### ISSUE TITLE
Include different options for region color palette

### ISSUE THREAD

**Issue by johannah-pik** [pik-piam/mip#51]
Depending on the use case, different color palettes are desired for the REMIND regions.
The new region colors from this PR are optimized for distinguishability (e.g. for the REMIND validation).
For publications, grouped colors for like-wise regions could be better. 

Here is a proposal from Nico for an grouped color palette, that could be implemented as an alternative choice:

OECD regions
plotstyle.add("EUR",  "EU", rgb(0.0, 0.0, 0.99, 1), marker = 23, replace = TRUE )
plotstyle.add("NEU",  "Other Europe", rgb(0.0, 0.0, 0.65,  1), marker = 9, replace = TRUE )
plotstyle.add("USA",  "USA",  rgb(0.5, 0.5, 0.99, 1), marker = 10, replace = TRUE )
plotstyle.add("JPN",  "Japan", rgb(0.0, 0.5, 0.99, 1), marker = 13, replace = TRUE )
plotstyle.add("CAZ",  "Can, Aus, NZ", rgb(0.3, 0.3, 0.8, 1), marker = 17, replace = TRUE )

Regions with developing and Emerging markets
plotstyle.add("CHA",  "China",      rgb(0.3, 0.99, 0.3, 1), marker = 19, replace = TRUE )
plotstyle.add("IND",  "India",      rgb(0.3, 0.65,  0.3, 1), marker = 8, replace = TRUE )
plotstyle.add("OAS",  "Other Asia", rgb(0.6, 0.99, 0.0, 1), marker = 11, replace = TRUE )
plotstyle.add("SSA",  "Sub-Sahra Africa", rgb(0.5, 0.0, 0.99, 1), marker = 15, replace = TRUE )
plotstyle.add("MEA",  "Middle East & N. Africa", rgb(0.7,  0.2, 0.99, 1), marker = 17, replace = TRUE )
plotstyle.add("LAM",  "Latin America", rgb(0.9, 0.6, 0.5, 1), marker = 12, replace = TRUE

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
State as of 0.140.1:
![image](https://user-images.githubusercontent.com/53254462/207557479-43ad5cc4-da00-4661-985b-96121c87ad81.png)


This merge request:
![image](https://user-images.githubusercontent.com/53254462/207557232-dffd5574-fbbd-4e2c-961f-9026bf15b0c7.png)

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
Clarified.  This is indeed a proposal, not a merge request.

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
> The option to choose a different palette (such as Nico's proposal) still has to be implemented sometime in the future, right?

Very much so, as `plotstyle.add()` is a destructive operation.

**Comment by nicobauer** [comment]
Somehow I remember that @dklein-pik was the master of plotstyle and the default csv file that is used. David, if so, could you up-date?

**Comment by nicobauer** [comment]
@0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q I only use plotstyle.add locally, where it does not affect the overall default values.

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
> @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q I only use plotstyle.add locally, where it does not affect the overall default values.

Yes.  But you can't go back to default values.  So there is no option for you to use either one or the other, except for manually setting them every time.  Which defeats the purpose of the plot styles to some degree.

**Comment by dklein-pik** [comment]
> Somehow I remember that @dklein-pik was the master of plotstyle and the default csv file that is used. David, if so, could you up-date?

I am not sure what you want. But if you want to update colors you can do it here https://github.com/pik-piam/mip/blob/master/inst/extdata/plotstyle.csv
